resources/updater.rb

```diff
@@ -39,11 +41,10 @@
 
 action_class.class_eval do
   def load_mixlib_install
-    require 'mixlib/install'
```

Comment:
So we can still just do the require and rescue it which avoids adding the gem resource on every converge. Check out how we used to do it in aws

https://github.com/chef-cookbooks/aws/commit/11a13022b3d25cb016e85e91c9c38906c0e0af49#diff-6d3244c03f3201830fac92ae7572037bL20